runtime/oti/j9modifiers_api.h

```diff
@@ -83,6 +83,7 @@
 #ifdef J9VM_OPT_VALHALLA_VALUE_TYPES
 /* Will need to modify this if ValObject/RefObject proposal goes through */
 #define J9ROMCLASS_IS_VALUE(romClass)	_J9ROMCLASS_SUNMODIFIER_IS_SET((romClass), J9AccValueType)
+#define J9ROMCLASS_IS_ATOMIC(romClass)	_J9ROMCLASS_SUNMODIFIER_IS_SET((romClass), J9AccAtomic)
```

Comment:
Extra spacing here.

Reply:
It is a tab after `(romClass)`, the same as the line above. It seems that tab is shown as 8 whitespaces in github view by default, so it looks like there are extra spaces.

Reply:
This is what it looks like in my eclipse:

![Capture](https://user-images.githubusercontent.com/18008382/126545341-1ee69f8f-3eba-4498-827d-f637e31b6533.JPG)

Reply:
OK

Reply:
I don't think we should be using tabs anywhere other than as a group at the beginning of a line. It should be a single space after the first `)`, but that sort of cleanup should be done separately.

Reply:
Agreed on both counts.